### PR TITLE
feat(Elife): Use link to executable for social media sharing

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: src/themes/elife/lib/downloads.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { after, append, create, select } from '../../../util'
 import { getArticlePdfUrl, getFiguresPdfUrl } from './dataProvider'
 import { articleData } from './query'
 
-const deriveUrl = (type: string, id: string, title = ''): string => {
+const deriveUrl = (type: string, id: string): string => {
```
Comment (nlisgo) [inline src/themes/elife/lib/downloads.ts:5]: This is unrelated but the 3rd parameter is no longer necessary.

**Comment by nlisgo** [comment]
This is a followup to https://github.com/stencila/thema/pull/231. This will be marked as `Ready for review` once that has been merged in.

**Review by discodavey — CHANGES_REQUESTED** [review]
Looks good but cannot see the executables when clicking the links in the Socials. Is this correct?

**Comment by nlisgo** [comment]
> Looks good but cannot see the executables when clicking the links in the Socials. Is this correct?

This is significant actually because the media card may not be too interesting for iframe content (not sure on this) but we can look at that after the launch? Currently, the executables are not publicly accessible so this is the main reason why the media card will not be great.

**Comment by discodavey** [comment]
@nlisgo That's what I thought. I will pass it on what I can see with the code then.

**Review by discodavey — APPROVED** [review]

**Comment by nlisgo** [comment]
@alex-ketch we can see this change yet: https://elife.stencila.io/article-30274/

**Comment by alex-ketch** [comment]
@nlisgo We added caching of the Thema version number to the Hub to avoid network redirects and improve page load times.
The changes from this PR are now visible at the above link (https://elife.stencila.io/article-30274/), and I'll log an issue in the Hub to make sure these changes are distributed faster.

**Comment by nlisgo** [comment]
I can't see the changes yet. I will let you know if it is still an issue tomorrow morning.

**Pull request merged**